.travis.yml

```diff
@@ -1,31 +0,0 @@
-os: linux
-language: node_js
-node_js:
-- '14'
-addons:
-  chrome: stable
-before_install: "source scripts/helpers.sh"
-install: yarn install
-before_script: "yarn build"
-script:
-  - "./scripts/travis.sh"
-dist: xenial
```

Comment:
Using `ubuntu-latest` seems to work without having to define a particular dist. Not even sure we could do it with github actions